c3/generator/generator.py

```diff
@@ -125,4 +125,8 @@ def generate_signals(self, instr: Instruction):
                 signal_stack.append(outputs)
             # The stack is reused here, thus we need to deepcopy.
             gen_signal[chan] = copy.deepcopy(signal_stack.pop())
+
+        # Hack to use crosstalk. Will be generalized to a post-processing module.
```

Comment:
Add a `# TODO` note here so this is not completely lost and can be picked up in later refactorings

Reply:
done